src/main/java/net/snowflake/client/jdbc/cloud/storage/SnowflakeAzureClient.java

```diff
@@ -202,8 +191,10 @@ public StorageObjectSummaryCollection listObjects(String remoteStorageLocation,
       Iterable<ListBlobItem> listBlobItemIterable =
           container.listBlobs(
               prefix, // List the BLOBs under this prefix
-              true // List the BLOBs as a flat list, i.e. do not list directories
-              );
+              true, // List the BLOBs as a flat list, i.e. do not list directories
```

Comment:
When the operationContext object containing the proxy information is passed as a parameter to the Azure blob functions that communicate with the Azure server, the connection is made via the proxy.